Src/Newtonsoft.Json/Converters/EntityKeyMemberConverter.cs

```diff
@@ -32,7 +32,7 @@
 namespace Newtonsoft.Json.Converters
 {
     /// <summary>
-    /// Converts an Entity Framework <c>EntityKey</c> to and from JSON.
+    /// Converts an Entity Framework <see cref="T:System.Data.EntityKeyMember"/> to and from JSON.
```

Comment:
The member ID syntax needs to stay in this instance, since the referred type is in an assembly (System.Data.Entity.dll) that isn't referenced by default.